README.md

```diff
@@ -320,6 +320,43 @@ $new_customer = $customer->createContact([
 ]);
 ```
 
+**List Customer Notes from a customer**
+```php
+$customer = ChartMogul\Customer::retrieve($uuid);
+$customer_notes = $customer->customer_notes([
+  'cursor' => 'aabbccdd...'
+]);
+```
+
+**Create a Customer Note from a customer**
+```php
+$customer = ChartMogul\Customer::retrieve($uuid);
+$new_customer_note = $customer->createCustomerNote([
+  'type' => 'note',
+  'text' => 'This is a note'
+]);
+```
+
+**Get a Customer Note from a customer**
+```php
+$customer = ChartMogul\Customer::retrieve($uuid);
+$customer_note = $customer->retrieveCustomerNote($note_uuid);
+```
+
+**Update a Customer Note from a customer**
+```php
+$customer = ChartMogul\Customer::retrieve($uuid);
+$updated_customer_note = $customer->updateCustomerNote($note_uuid, [
+  'text' => 'This is a new note'
+]);
+```
+
+**Delete a Customer Note from a customer**
+```php
+$customer = ChartMogul\Customer::retrieve($uuid);
+$updated_customer_note = $customer->deleteCustomerNote($note_uuid)
```

Comment:
```suggestion
$result = $customer->deleteCustomerNote($note_uuid) # '{}'
```

See how other endpoints do it for the delete response (dev.chartmogul.com [docs](https://dev.chartmogul.com/reference/delete-a-contact))

<details><summary>Details</summary>
<p>

![Screenshot 2023-12-11 at 11 17 01 AM](https://github.com/chartmogul/chartmogul-php/assets/8046636/37cab935-7877-4512-99b4-9cdb914d7358)


</p>
</details>